spec/features/steps/admin/site_search_steps.rb

```diff
@@ -3,13 +3,12 @@
 module Admin
   module SiteSearchSteps
     def it_should_have_added_the_page_to_search(contact)
-      # Check that it's being added as the correct document type
+      # Checks that it's being added as the correct document type
       rummager_presenter = ContactRummagerPresenter.new(contact)
-      expected_json = rummager_presenter.present.merge(
-        _type: 'contact',
-      )
-      # export to JSON so that the public_timestamp timestamp is in a string and can
-      # be compared to the JSON captured by Webmock
+      expected_json = rummager_presenter.present.merge(_type: 'contact')
+
+      # exports to JSON so that the public_timestamp timestamp
+      # is in a string and can be compared to the JSON captured by Webmock
       expected_json = JSON.parse(expected_json.to_json)
```

Comment:
I think this statement is no longer necessary since the `.iso8601` call in `contact_rummager_presenter.rb`.